daemons/controld/controld_execd_state.c

```diff
@@ -90,14 +90,6 @@ fail_pending_op(gpointer key, gpointer value, gpointer user_data)
     return TRUE;
 }
 
-gboolean
-lrm_state_is_local(lrm_state_t *lrm_state)
-{
-    return (lrm_state != NULL)
-           && pcmk__str_eq(lrm_state->node_name, controld_globals.our_nodename,
-                           pcmk__str_casei);
-}
-
```

Comment:
This was here for logical separation -- controld_execd_state.c contains accessors for lrm_state_t (there are some borderline accessors in controld_execd.c currently, but we don't need to add to them)

controld_execd.c/controld_execd_state.c could benefit from a close look at how the files should be split, but for now I'd leave it